File: Java/String/program18.java
//Accept the string from user and display that string in lowercase

import java.util.*;

class Stringx
{
    public String str;

    public void Accept()
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the string");
        str = sobj.nextLine();
    }

    public void strlwrX()
    {
        System.out.print("String in lower case : "+str.toLowerCase());        
    }
}

class program18
{
    public static void main(String arg[])
    {
        Stringx obj = new Stringx();

        obj.Accept();

        obj.strlwrX();
    }
}

File: C/String/program18.c
//Accept the string from user and display that string in uppercase

#include<stdio.h>

void struprX(char str[])
{
    while(*str != '\0')
    {
        if((*str >= 'a') && (*str <= 'z'))
        {
            *str = *str - 32;
        }
        str++;
    }
}

int main()
{
    char Arr[20];

    printf("Enter string: \n");
    scanf("%[^'\n']s",Arr);

    struprX(Arr);
    printf("String in upper case is : %s ",Arr);

    return 0;
}
